SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceEventBuilder.cpp

```diff
@@ -100,19 +100,23 @@ FSpatialTraceEvent FSpatialTraceEventBuilder::GetEvent() &&
 	return MoveTemp(SpatialTraceEvent);
 }
 
-FSpatialTraceEvent FSpatialTraceEventBuilder::CreateProcessRPC(const UObject* Object, UFunction* Function)
+FSpatialTraceEvent FSpatialTraceEventBuilder::CreateProcessRPC(const UObject* Object, UFunction* Function,
+															   const EventTraceUniqueId& UniqueTraceId)
```

Comment:
Did we try using the Send SpanId for the UniqueTraceId?

Reply:
No but only because Span ids have changed to an opaque pointer instead in an upcoming worker release - it's possible they might have also added a nice generator though

Reply:
Oh yeah I remember you explaining yesterday. My bad.

Reply:
We could use it until the worker change comes, although if they don't have the generator we might have to reimplement it but maybe that's not so bad